src/components/DateRangePicker.jsx

```diff
@@ -43,6 +43,7 @@ const defaultProps = {
   numberOfMonths: 2,
   showClearDates: false,
   disabled: false,
+  hidePickerOnClear: false,
```

Comment:
If you click the clear button now when the datepicker is open, I don't think this will actually hide the picker so this prop name is a misnomer. Something like `dontReopenPickerOnClear` maybe makes more sense... or even in this case, it might be worth it to have the semver breaking change of `reopenPickerOnClear`. What do you think @ljharb?